package.json

```diff
@@ -1,60 +1,28 @@
 {
   "name": "scratch-blocks",
-  "version": "1.1.86",
+  "version": "2.0.0",
   "description": "Scratch Blocks is a library for building creative computing interfaces.",
   "author": "Massachusetts Institute of Technology",
   "license": "Apache-2.0",
-  "homepage": "https://github.com/scratchfoundation/scratch-blocks#readme",
+  "homepage": "https://github.com/gonfunko/scratch-blocks#readme",
   "repository": {
     "type": "git",
-    "url": "https://github.com/scratchfoundation/scratch-blocks.git"
+    "url": "https://github.com/gonfunko/scratch-blocks.git"
   },
-  "main": "./dist/vertical.js",
-  "browser": "./shim/vertical.js",
+  "main": "./dist/main.js",
+  "private": true,
   "scripts": {
-    "deploy": "rimraf gh-pages/closure-library/scripts/ci/CloseAdobeDialog.exe && gh-pages -t -d gh-pages -m \"Build for $(git log --pretty=format:%H -n1) [skip ci]\"",
-    "prepare": "husky install",
-    "prepublish": "python build.py && webpack",
-    "test": "npm run test:messages && npm run test:unit",
-    "test:lint": "eslint .",
```

Comment:
Could you keep this + the eslint dependency? I'd rather we just keep linting how they want than to have to fix it laterz :P

Reply:
Done!

Reply:
Sweet, but as discussed offline, we're not going to lint until the outstanding changes are in, just to be more efficient.